### PR TITLE
Adds Contractor ERT

### PR DESCRIPTION
**Pull request by CapCamIII**
## About The Pull Request

First off before anyone yells at me over the contrib guidelines, stan gave me the okay to PR and monkey said the lore was fine.

Now, onto what this does:
Adds a new marine-allied ERT, Vanguard's Arrow Incorporated Private Military Contractors, they can arrive during random distress signals or be sent by admins.
There are 3 versions of the team: Standard team with 7 contractors and potential synthetic replacing the seventh, whom can be randomly chosen by a distress signal, Platoon which is a a group of 28 contractors and 2 synthetics; only can arrive via admins, and covert which is an admin-spawned elite team, 7 with the potential of the seventh being replaced with a synthetic, whom come with NVGs,
Adds a new tactical variant of MAR30, MAR40, MAR50, MK221, MK2, MK1, and M1911 which comes standard with suppressors and mag-harns, with as well as ammo-counters.
M4A3 bullets made into regular pistol bullets to allow heavy pistol bullets to be buffed without effecting marine weapons, as a side effect this means M4A3 can exchange bullets with beretta and M43 now.
Regular pistol bullets given old heavy-pistol bullet stats(pre-buff) to not nerf the M4A3, as a side effect this buffed the beretta and M43(CLF pistol)
Heavy pistol bullets buffed to 55 dam(from 40) and 15 AP(from 5 or 0), this effects the M1911, KT-42, automag, and survivor deagle.
M1911 buffed with an extra 5 rounds per magazine and extra fire-rate on top of the heavy pistol buff.

Lore-wise these guys are American Mil-Contractors, used to be the best of the best in the American PMC market, bit washed up now but kept afloat through an ongoing contract against Weyland-Yutani, though at the cost of pissing WY(and by extension the TWE) off BADLY.
They are split into five sections, three of which are represented in the PR: VAICC(Central Command), VAISP(Special Protection), VAISO(Special Operations, in the PR as covert ERT), VAIPO(Primary Operations, in PR as standard and platoon ERT), VAIMS(Medical-Science, in the PR as medics for the teams). The team that responds to the distress colony is from the USCSS Inheritor, part of Task-Force Charlie, VAIPO.

Plan is to make a second PR after this one is merged, making use of the AP sprites added, and adding the rest of the contractor types from the lore of the ERT, adding HUDs for them.

Images of them: https://imgur.com/a/h5pFFUA

## Why It's Good For The Game

New marine-allied ERT who look awesome and serve as a more lore-friendly way of admins sending reinforcements than a magically appearing USCM vessel.
More roles for events, specifically a marine-allied(and anti-WY) set of roles.
M4A3 can interchange bullets with beretta and M43
M1911(and surv-deagle/kt42/automag) have better damage and in the case of M1911; better fire-rate and mag-size too.

## Changelog

:cl: CaptainCamo
add: Added a new marine-allied ERT, Military Contractors!
add: Military Contractors are a marine allied ERT, with a 7 man team, including synthetic in ERT pool, with an additional 28 platoon or 7 covert teams for admin-spawn.
add: Contractors come standard with suppressed rifles, M1911s, ceramic plates, M3-VL, t-shirts, and shades.
add: Presets, headset, and comms channel for above.
add: Variants of the MAR40, MAR30, MAR50, MK2, MK1, MK221, and M1911 with preset attachments and ammo-counters.
add: M4A3 can interchange bullets with Beretta and M43.
balance: Heavy pistol bullets buffed to 15 AP-55 damage, note: this does not effect M4A3
balance: M1911 given additional fire-rate
balance: CO deagle ammo given an extra 10 damage to keep it superior to surv deagle ammo.
imageadd: Added unused MAR AP magazine sprites.
/:cl:

### CODE REVIEW

**Comment by Superreallycoolguy** [comment]
Honestly this looks like it's doing way too much way too soon. Like some of their kits seem iffy, some really powerful (im a plate hater). There's a buncha balances for an ERT being added. 

New stuff is cool but this is just unnecessary, ill be the one to say it, it doesn't have a niche, its just a pmc ERT with marine flavor, they're not even unique in that right, they used cannibalized uniforms and armor of other ERTs/Jobs.

Like...the "covert" variant just use MARSOC gear, that's really lazy in my opinion. I think this can be a really cool thing! Just don't half ass it, get custom sprites, find their place, and idk just make it feel more fitting than a "WOAH!!!! NEW COOL ERT!!!!".

Sorry dude but I'm super against this

**Comment by CapCamIII** [comment]
To address your points: @Superreallycoolguy 
Plates: They have nearly no armor from FF, and are made up of completely ballistic weapons(causes massive FF usually).

Balance: They are kind of meant to be glass-cannons, both in HvH and in HvX, HvH wise plates were nerfed into the ground, blocking only around 4 bullets before dropping, in both HvH and HvX, the M3-VL vest(which is basically a normal ballistic vest with a light, storage slot, and mag-harn capability) has quite poor armor. The idea I had with these guys is a wall of bullets focusing on the MAR50s with bipods, which is why they all have a lot of ammo and MARs(which are quite powerful to be honest), I wouldn't really call anything iffy or too OP; perhaps with the exception of the extra magazines I gave the MGs, but you were rather vague so it would be nice if you elaborated on that.

On the niche thing, I agree it doesn't have a niche, but theres not enough ERTs for niches to be exist IMO, the last ERT added to the random pool was feral xenos; which are in my IMO just XRT; and the last actual ERT was added before I even started playing in late 2019.

On the lack of uniqueness: I disagree, these guys have a lot of gear from different factions yes, but I did that for a reason, I based what they have off of IRL images of PMCs and such in the modern day, see the light-vests, t-shirts, MARs(basically AKs), they have a lot of stuff from other groups, but it is put together in such a way that makes it unique. On the re-skinned PMCs part, are freelancers not re-skinned PMCs with worse gear, and even then: they are both a bit rarer than PMCs.

On the covert varient using MARSOC gear, I did that to establish the look that they had(and to lessen the required workload, I do not want to make a new subtype of helmet just to give it a unique name), and even then its 2(three if you count the NODs), pieces of gear: the helmet and balaclava: either of which could easily be surplus IC, and with their full set of gear there isn't much of a chance of mistaking the two. Getting someone to sprite something is rather hard, and why do that when you have the tools already available to you., so I mixed together gear to give them a unique look and feel, even if it re-uses things from other jobs/ERTs.

Examples on the IRL images of military contractors and such I based these guys on: https://imgur.com/a/uIG56cc

**Comment by CapCamIII** [comment]
i'll deal with the conflicts tommorow if I can.

**Comment by stanalbatross** [comment]
fix conflict

**Comment by MusselMain** [comment]
I have to point out that the 1911 buffs you suggested should altered slightly, as they could potentially conflict with the LV522 map (as 1911's are very common there, and are already getting a buff in that PR), otherwise this is a cool idea. I also suggest you figure out how this faction's relations with other factions will look like (if you haven't already), I also really appreciate the KT-42 Automag buff, the poor thing had the damage of a M41A MK2 round, a firerate worse than the deagle, very rare ammo, and somehow was forced upon survivors, CLF SL's, and Freelance Warlords, this will allow it to be at least decent instead of the barely stronger than the CMB that it currently is.

**Comment by CapCamIII** [comment]
> I have to point out that the 1911 buffs you suggested should altered slightly, as they could potentially conflict with the LV522 map (as 1911's are very common there, and are already getting a buff in that PR), otherwise this is a cool idea.

If it does conflict, its a 1 line change for either of us, so I'm banking on my changes getting in first.


>I also suggest you figure out how this faction's relations with other factions will look like (if you haven't already),

Faction relations are as follows: WY: Hostile, but not overtly, USCM: Allied, CLF: Hostile, UPP: Same as USCM, Freelancers: Neutral or hostile depending on freelancers being hostile to USCM.


>I also really appreciate the KT-42 Automag buff, the poor thing had the damage of a M41A MK2 round, a firerate worse than the deagle, very rare ammo, and somehow was forced upon survivors, CLF SL's, and Freelance Warlords, this will allow it to be at least decent instead of the barely stronger than the CMB that it currently is.

Yeah, its still not going to be that good comparatively, but its not directly worse now.

@MusselMain

**Comment by CapCamIII** [comment]
Only thing left til un-drafting is fixing commit count, which I'll find out how to do tomorrow.

**Comment by c4xmaniac** [comment]
Does the synth in the ERT have a check to make sure only WL holders can get it?

**Comment by CapCamIII** [comment]
Synth WL check was copied from the other ERTs, assuming the WL check for those work; this one will as well. @c4xmaniac

File: code/datums/emergency_calls/contractor.dm [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+	max_medics = 2
+	max_heavies = 2
+	var/max_synths = 1
+	var/synths = 0
+
+
+/datum/emergency_call/contractors/New()
+	..()
+	arrival_message = "[MAIN_SHIP_NAME], this is USCSS Inheritor with Vanguard's Arrow Incorporated, Primary Operations; we are responding to your distress call and boarding in accordance with the Military Aid Act of 2177, authenticication code Lima-18153. "
+	objectives = "Assure the survival of the [MAIN_SHIP_NAME], eliminate any hostiles, and assist the crew in any way possible."
```
Comment (stanalbatross) [inline code/datums/emergency_calls/contractor.dm:16]:
```suggestion
	objectives = "Ensure the survival of the [MAIN_SHIP_NAME], eliminate any hostiles, and assist the crew in any way possible."
```

File: code/datums/emergency_calls/contractor.dm [hunk 2]
```diff
@@ -0,0 +1,148 @@
+/datum/emergency_call/contractors
+	name = "Military Contractors (Squad)"
```
Comment (stanalbatross) [inline code/datums/emergency_calls/contractor.dm:2]:
```suggestion
	name = "Military Contractors (Squad) (Friendly)"
```

File: code/datums/emergency_calls/contractor.dm [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+	to_chat(M, SPAN_BOLD("You are employed by Vanguard's Arrow Incorporated(VAI), as a member of VAI Primary Operations(VAIPO)"))
+	to_chat(M, SPAN_BOLD("You are stationed on-board the USCSS Inheritor, a part of VAIPO Task-Force Charlie."))
+	to_chat(M, SPAN_BOLD("Under the directive of the VAI executive board, you have been assist in riot control, military aid, and to assist USCMC forces wherever possible."))
+	to_chat(M, SPAN_BOLD("The USCSS Inheritor is staffed with crew of roughly three hundred military contractors, and fifty support personnel."))
+	to_chat(M, SPAN_BOLD("Assist the USCMC Force of the [MAIN_SHIP_NAME] however you can."))
+	to_chat(M, SPAN_BOLD("As a side-objective, VAI has been hired by an unknown benefactor to engage in corporate espionage and sabotage against Weyland-Yutani, avoid direct conflict; you aren't VAISO; but attempt to recover Wey-Yu secrets and plans if possible."))
+
+
+/datum/emergency_call/contractors/platoon
+	name = "Military Contractors (Platoon)"
```
Comment (stanalbatross) [inline code/datums/emergency_calls/contractor.dm:78]:
```suggestion
	name = "Military Contractors (Platoon) (Friendly)"
```

File: code/datums/emergency_calls/contractor.dm [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+	mob_min = 12
+	mob_max = 41
+	probability = 0
+	max_medics = 5
+	max_heavies = 4
+	max_engineers = 3
+
+/datum/emergency_call/contractors/covert
+	name = "Military Contractors (Covert)"
```
Comment (stanalbatross) [inline code/datums/emergency_calls/contractor.dm:87]:
```suggestion
	name = "Military Contractors (Covert) (Hostile to WY)"
```

File: code/datums/skills.dm [hunk 5]
```diff
@@ -1250,6 +1250,94 @@ Private Military Contractors
...
+		SKILL_CONSTRUCTION = SKILL_CONSTRUCTION_ENGI,
+		SKILL_FIREARMS = SKILL_FIREARMS_MAX,
+		SKILL_POLICE = SKILL_POLICE_SKILLED,
+		SKILL_VEHICLE = SKILL_VEHICLE_CREWMAN,
+		SKILL_MEDICAL = SKILL_MEDICAL_TRAINED,
+		SKILL_MELEE_WEAPONS = SKILL_MELEE_TRAINED,
+		SKILL_JTAC = SKILL_JTAC_BEGINNER,
+		SKILL_ENDURANCE = SKILL_ENDURANCE_TRAINED,
+		SKILL_POWERLOADER = SKILL_POWERLOADER_MASTER,
+
```
Comment (stanalbatross) [inline code/datums/skills.dm:1272]:
```suggestion
```

**Review by stanalbatross — CHANGES_REQUESTED** [review]

**Review by stanalbatross — CHANGES_REQUESTED** [review]
balance stuff:

- standards should not get only extended mags by default. They can get a few but not tons.
- standards should only get 2 nades and 1 breaching charge
- give them all ERT first aid pouches instead of default ones
- SOCOM m1911 needs a unique sprite (idc if it's a palette swap but it needs one)
- why do they only get MARs? add some weapon variety such as shotguns
- this is too many drums
image
- no OP defib for the synth
image
- no, no, and double no on the nades for SL. give them a pistol belt and nades in pouch/webbing if you want
image
- SLs should get level 1 medic skill and only sensorhuds not medhuds
Mirror these changes for the coverts

Also, no, no and quadruple no on 11 mob spawns for the ERT. Even CLF, the weakest ERT who are meant to be made up for by high numbers, only get 10.

Mob max for the ERT should be seven - one engi, one medic, and one heavy or synth.

**Comment by MusselMain** [comment]
On Aug 9, 2022, at 11:00 AM, Stan_Albatross ***@***.***> wrote:

﻿

@stanalbatross requested changes on this pull request.

________________________________

In code/datums/emergency_calls/contractor.dm<https://github.com/cmss13-devs/cmss13/pull/786#discussion_r941450002>:

> +/datum/emergency_call/contractors
+       name = "Military Contractors (Squad)"
+       mob_max = 11
+       probability = 15
+
+       max_engineers =  1
+       max_medics = 2
+       max_heavies = 2
+       var/max_synths = 1
+       var/synths = 0
+
+
+/datum/emergency_call/contractors/New()
+       ..()
+       arrival_message = "[MAIN_SHIP_NAME], this is USCSS Inheritor with Vanguard's Arrow Incorporated, Primary Operations; we are responding to your distress call and boarding in accordance with the Military Aid Act of 2177, authenticication code Lima-18153. "
+       objectives = "Assure the survival of the [MAIN_SHIP_NAME], eliminate any hostiles, and assist the crew in any way possible."


⬇️ Suggested change

-       objectives = "Assure the survival of the [MAIN_SHIP_NAME], eliminate any hostiles, and assist the crew in any way possible."
+       objectives = "Ensure the survival of the [MAIN_SHIP_NAME], eliminate any hostiles, and assist the crew in any way possible."


________________________________

In code/datums/emergency_calls/contractor.dm<https://github.com/cmss13-devs/cmss13/pull/786#discussion_r941450256>:

> @@ -0,0 +1,148 @@
+/datum/emergency_call/contractors
+       name = "Military Contractors (Squad)"


⬇️ Suggested change

-       name = "Military Contractors (Squad)"
+       name = "Military Contractors (Squad) (Friendly)"


________________________________

In code/datums/emergency_calls/contractor.dm<https://github.com/cmss13-devs/cmss13/pull/786#discussion_r941450984>:

> +     else
+               to_chat(M, SPAN_BOLD("You were brought online in a civilian factory."))
+               to_chat(M, SPAN_BOLD("You were programmed with all of the medical and engineering knowledge a military fighting force support asset required."))
+               to_chat(M, SPAN_BOLD("You were soon after bought by the Vanguard's Arrow Security Consulting Agency to act as support personnel."))
+               to_chat(M, SPAN_BOLD("Some months after your purchase, you were assigned to the USCSS Marlow, a VASC transport vessel."))
+       to_chat(M, SPAN_BOLD("You are [pick(80;"unaware", 15;"faintly aware", 5;"knowledgeable")] of the xenomorph threat."))
+       to_chat(M, SPAN_BOLD("You are employed by Vanguard's Arrow Incorporated(VAI), as a member of VAI Primary Operations(VAIPO)"))
+       to_chat(M, SPAN_BOLD("You are stationed on-board the USCSS Inheritor, a part of VAIPO Task-Force Charlie."))
+       to_chat(M, SPAN_BOLD("Under the directive of the VAI executive board, you have been assist in riot control, military aid, and to assist USCMC forces wherever possible."))
+       to_chat(M, SPAN_BOLD("The USCSS Inheritor is staffed with crew of roughly three hundred military contractors, and fifty support personnel."))
+       to_chat(M, SPAN_BOLD("Assist the USCMC Force of the [MAIN_SHIP_NAME] however you can."))
+       to_chat(M, SPAN_BOLD("As a side-objective, VAI has been hired by an unknown benefactor to engage in corporate espionage and sabotage against Weyland-Yutani, avoid direct conflict; you aren't VAISO; but attempt to recover Wey-Yu secrets and plans if possible."))
+
+
+/datum/emergency_call/contractors/platoon
+       name = "Military Contractors (Platoon)"


⬇️ Suggested change

-       name = "Military Contractors (Platoon)"
+       name = "Military Contractors (Platoon) (Friendly)"


________________________________

In code/datums/emergency_calls/contractor.dm<https://github.com/cmss13-devs/cmss13/pull/786#discussion_r941451157>:

> +     to_chat(M, SPAN_BOLD("The USCSS Inheritor is staffed with crew of roughly three hundred military contractors, and fifty support personnel."))
+       to_chat(M, SPAN_BOLD("Assist the USCMC Force of the [MAIN_SHIP_NAME] however you can."))
+       to_chat(M, SPAN_BOLD("As a side-objective, VAI has been hired by an unknown benefactor to engage in corporate espionage and sabotage against Weyland-Yutani, avoid direct conflict; you aren't VAISO; but attempt to recover Wey-Yu secrets and plans if possible."))
+
+
+/datum/emergency_call/contractors/platoon
+       name = "Military Contractors (Platoon)"
+       mob_min = 12
+       mob_max = 41
+       probability = 0
+       max_medics = 5
+       max_heavies = 4
+       max_engineers = 3
+
+/datum/emergency_call/contractors/covert
+       name = "Military Contractors (Covert)"


⬇️ Suggested change

-       name = "Military Contractors (Covert)"
+       name = "Military Contractors (Covert) (Hostile to WY)"


________________________________

In code/datums/skills.dm<https://github.com/cmss13-devs/cmss13/pull/786#discussion_r941451577>:

> +*/
+/datum/skills/contractor
+       name = "Contractor Standard"
+       skills = list(
+               SKILL_CQC = SKILL_CQC_MASTER,
+               SKILL_ENGINEER = SKILL_ENGINEER_ENGI,
+               SKILL_CONSTRUCTION = SKILL_CONSTRUCTION_ENGI,
+               SKILL_FIREARMS = SKILL_FIREARMS_MAX,
+               SKILL_POLICE = SKILL_POLICE_SKILLED,
+               SKILL_VEHICLE = SKILL_VEHICLE_CREWMAN,
+               SKILL_MEDICAL = SKILL_MEDICAL_TRAINED,
+               SKILL_MELEE_WEAPONS = SKILL_MELEE_TRAINED,
+               SKILL_JTAC = SKILL_JTAC_BEGINNER,
+               SKILL_ENDURANCE = SKILL_ENDURANCE_TRAINED,
+               SKILL_POWERLOADER = SKILL_POWERLOADER_MASTER,
+


⬇️ Suggested change

-

I could be wrong but I feel like those skills seem a bit busted for a standard in a ERT, seems a bit strange balance wise that they would be so versatile (even when compared to MARSOC or spec ops).

**Comment by CapCamIII** [comment]
@MusselMain You should probably fix your comment, its way too big.
Basically, they are meant to be very versatile; able to do most things, per skill:
CQC: Grabbing and disarming, no real combat use, based on MP skill.
Engineer: Survivor level engi(able to hack doors and repair plasteel cades), allows them to do most everything at a basic level.
Construction: Plasteel cade construction, only engi has mats; but if they are in an area with spare mats, they can all help out.
Firearms: PMC level firearms skill.
Police: PMC/MP level skill, tasers and baton use.
Vehicle: Bit iffy yes, but they cannot GET vehicles unless an admin spawns them, as well as I am likely going to do a PR after this following up that includes things such as a VC role(and removing VC skills from standards).
Medical: Injector use, no pills or medic-hud.
Melee: Not very useful, but no reason not to, equal to spec or SL; forgot which exactly.
JTAC: Faster JTAC lases for their rangefinders, no use in hijack but as marine reinforcements admin-sent this would be great.
Endurance: Level 2 endurance, just above marines.
Powerloader: Why not?

**Comment by CapCamIII** [comment]
alright i did the changes but i messed something up, i'll fix what I can later, been doing this all day.

**Review by stanalbatross — CHANGES_REQUESTED** [review]
- give vaiso SL a sensormate, they can't use healthmates
- put the new VAI radio channel in constants.js

**Comment by stanalbatross** [comment]
the rest is fine

**Review by stanalbatross — APPROVED** [review]
code approved

**Review by Agameofscones — CHANGES_REQUESTED** [review]
sprites are good save for the AP mags. Do strong bold stripes and not the smaller little stripes you have. The m41a mags are a good example of what I mean. Right now the AP is hard to identify at a glance.

**Comment by Agameofscones** [comment]
fix icon conflict, will re-review.

**Comment by CapCamIII** [comment]
@Agameofscones conflicts fixed :D

**Review by Agameofscones — APPROVED** [review]
hyup, guess that'll do

**Comment by MusselMain** [comment]
On Aug 9, 2022, at 6:59 PM, riot ***@***.***> wrote:

﻿

@MusselMain<https://github.com/MusselMain> You should probably fix your comment, its way too big.
Basically, they are meant to be very versatile; able to do most things, per skill:
CQC: Grabbing and disarming, no real combat use, based on MP skill.
Engineer: Survivor level engi(able to hack doors and repair plasteel cades), allows them to do most everything at a basic level.
Construction: Plasteel cade construction, only engi has mats; but if they are in an area with spare mats, they can all help out.
Firearms: PMC level firearms skill.
Police: PMC/MP level skill, tasers and baton use.
Vehicle: Bit iffy yes, but they cannot GET vehicles unless an admin spawns them, as well as I am likely going to do a PR after this following up that includes things such as a VC role(and removing VC skills from standards).
Medical: Injector use, no pills or medic-hud.
Melee: Not very useful, but no reason not to, equal to spec or SL; forgot which exactly.
JTAC: Faster JTAC lases for their rangefinders, no use in hijack but as marine reinforcements admin-sent this would be great.
Endurance: Level 2 endurance, just above marines.
Powerloader: Why not?

I apologize for the previous, I’m using a different computer that seems to be so old that it’s functions are significantly worse than my usual.

I am well aware of what the skills are, I was mostly talking about the vehicle skill, power loading skill, endurance skill and JTAC skill.

Having a vehicle skill can be pretty useful and in the current version of the game is limited to only a few roles, I feel like giving it to standards seems a bit silly.

Power loading is another rather rare skill that I once again think should be limited to anything but standards in this new ERT (like engineer or SL), as it’ll mean they can replace a fair amount of other roles if this is ever used as a marine reinforcement.

Endurance I have no problems with but this squad was described as “glass cannons” and giving them increased endurance doesn’t really Align with that statement (but it’s not a big deal if it’s slightly different).

JTAC makes sense for SL’s and RTO’s but if it’s standard among a whole squad who’s to say it won’t be abused (yes I’ve seen this happen before), taking the risk out of using rangefinders, and as you said, it’s not going to help in most scenarios so why add it, unless the idea is to make a weaker MARSOC with more versatility, I think it’s a bit strange to give them skills they won’t use or that will give them a massive advantage, it seems like in its current shape this ERT is a bit too strong compared to other squads (more mobs than CLF, SL has a ton of grenades, standards have only extended magazines, exclusively MAR’s which are fairly strong weapons, and despite being called glass cannons, carry M3-VL’s which while weak against acid are very good for PVP due to high ballistic armor). I don’t think this squad needs a whole redo, it’s a cool idea, but it needs a bit of work for sure.
Reminder that this is strictly my opinion and that I’m only basing it off my experience and knowledge (across 5 years of playing this game).

**Comment by CapCamIII** [comment]
Time to respond, going bit by bit.

> Having a vehicle skill can be pretty useful and in the current version of the game is limited to only a few roles, I feel like giving it to standards seems a bit silly.

Up to change if asked by a dev, what I meant by versatility, is these guys are meant to be able to pick up and use anything, excluding spec weapons, they find, that including vehicles.


> Power loading is another rather rare skill that I once again think should be limited to anything but standards in this new ERT (like engineer or SL), as it’ll mean they can replace a fair amount of other roles if this is ever used as a marine reinforcement.

Above.


> Endurance I have no problems with but this squad was described as “glass cannons” and giving them increased endurance doesn’t really Align with that statement (but it’s not a big deal if it’s slightly different).

Endurance is different than actual armor, it prevents fractures and such but doesn't actually stop you from dying as fast.


> JTAC makes sense for SL’s and RTO’s but if it’s standard among a whole squad who’s to say it won’t be abused (yes I’ve seen this happen before), taking the risk out of using rangefinders, and as you said, it’s not going to help in most scenarios so why add it

Again, meant to be able to use whatever they find effectively.

_______________________________________________________

Onto the next section, I changed a LOT due to preliminary balance review, most of what you said is outdated.
Albeit I have forgotten to update the PR description, I will do that now.


> more mobs than CLF

Same as PMCs now.


> SL has a ton of grenades

SL now only has five UGL grenades and 2 hand-grenades.


> standards have only extended magazines

Standards no longer have only extended magazines.


> exclusively MAR’s which are fairly strong weapons

Standards can now have a 40% chance of having a MAR40, 30% chance of having an M41A MK2, 20% chance of having an MK221, and 10% chance of having an M41A MK1


> carry M3-VL’s which while weak against acid are very good for PVP due to high ballistic armor

HvH is not an actual balance concern, although I did take it in mind with some portions of their equipment.

@MusselMain

**Pull request merged**